ui/plugins/controller.js

```diff
@@ -334,6 +334,19 @@ treeherder.controller('PluginCtrl', [
             }
         };
 
+        $scope.retriggerJobs = function(jobs) {
+            // Initial promise for sequential execution.
+            var jobPromise = Promise.resolve();
+            jobs.forEach(function(job) {
+                jobPromise = jobPromise.then(function() {
+                    return selectJob(job.id);
```

Comment:
I need to call selectJob() function to set the $scope.job and it has to be sequential to send correct job_ids to retriggerJob(). I know it is a bit hacky, so let me know if there is a better way.